docs/references/express/overview.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Clerk Express SDK
+description: Learn how to integrate Express into your Clerk application.
+---
+
+Clerk makes it simple to add authentication to your Express application. This documentation covers the capabilities and methods available from Clerk's Express SDK.
+
+See the [quickstart](/docs/quickstarts/express) to get started.
+
+## Available methods
+
+### `clerkMiddleware()`
+
+The `clerkMiddleware()` helper integrates Clerk authentication into your Express application. It is required to be set in the middleware chain before using other Clerk utilities, such as `requireAuth` and `getAuth()`.
+
+```js
+import { clerkMiddleware } from '@clerk/express'
+
+const app = express()
+
+// Pass no parameters
+app.use(clerkMiddleware())
+
+// Pass a function that will run as middleware
+app.use(clerkMiddleware(handler))
+
+// Pass options
+app.use(clerkMiddleware(options))
+
+// Pass both
+app.use(clerkMiddleware(handler, options))
+```
+
+#### Options
+
+The `clerkMiddleware()` function accepts [these options](https://clerk.com/docs/references/nextjs/clerk-middleware#clerk-middleware-options) plus the following:
+
+<Properties>
+  - `clerkClient`
+  - [`ClerkClient`](/docs/references/backend/overview#create-clerk-client-options)
+
+  An instance of the ClerkClient class. This is used to interact with the Clerk API.
+
+  ---
+
+  - `debug`
+  - `boolean`
+
+  A flag to enable debug mode. When enabled, the middleware will log debug information to the console.
+
+  ---
+
+  - `enableHandshake`
+  - `boolean`
+
+  Enables Clerk's handshake flow, which helps verify the session state when a session JWT has expired. It issues a 307 redirect to refresh the session JWT if the user is still logged in.
+</Properties>
+
+### `requireAuth`
+
+`requireAuth` is a middleware function that you can use to protect routes in your Express application. This function checks if the user is authenticated, and sends an error if they are not. If you want to customize the error response, you can handle it in an error middleware, as shown in the following example.
+
+`clerkMiddleware()` is required to be set in the middleware chain before this util is used.
+
+```js
+import { clerkMiddleware, requireAuth, UnauthorizedError } from '@clerk/express'
+import express from 'express'
+
+const app = express()
+
+const port = 3000
+
+// Apply centralized middleware
+app.use(clerkMiddleware())
+
+// Define a protected route
+app.get('/protected', requireAuth, (req, res) => {
+  res.send('This is a protected route')
+})
+
+// Customize the error response through an error middleware
+app.use((err, req, res, next) => {
+  // Use the provided UnauthorizedError class to check if
+  // the error is an unauthorized error.
+  if (err instanceof UnauthorizedError) {
+    return res.status(401).send('Unauthorized')
```

Comment:
I feel like maybe returning a redirect to a sign in page would be a more relevant example here